### PR TITLE
JavasScript validation hack + tabs to spaces

### PR DESCRIPTION
**Pull request by petrvecera**
We still should run it trough some beautifier

### CODE REVIEW

File: fileUpload/index.php [hunk 1]
```diff
@@ -3,20 +3,37 @@
...
+            lineColor: '#1a7269'
+          });
+          document.getElementById("uploadBtn").disabled = true;
+          fileInput = document.getElementById("inputFile")
+          urlInput = document.getElementById("url")
+        }); 
+
+        function validateField(){
+            var x = urlInput.className;
+            if(fileInput.value || (urlInput.value && (x.indexOf('invalid') == -1 ))){
```
Comment (petrvecera) [inline fileUpload/index.php]: Why does it have red highlight?

File: fileUpload/index.php [hunk 2]
```diff
@@ -3,20 +3,37 @@
...
+        <script type="text/javascript"> 
+        var fileInput = null,
+            urlInput = null;
+
+        $(document).ready(function() {
+          $('#upload').particleground({
+            dotColor: '#1a7269',
+            lineColor: '#1a7269'
+          });
+          document.getElementById("uploadBtn").disabled = true;
```
Comment (cervenyprofiq) [inline fileUpload/index.php]: Remove this row - Console error and page crash with it.

File: fileUpload/index.php [hunk 3]
```diff
@@ -138,36 +155,36 @@ function HumanSize($Bytes)
...
+                    </div>
+                </div>
+                <div class="row">
+                    <div class="input-field col s12">
+                        <input name="url" id="url" type="url" class="validate" oninput="validateField()">
+                        <label for="url">URL</label>
+                    </div>
+                </div>
+                <br/>
+                <button id="uploadBtn" type="submit" name="action" class="waves-effect waves-light btn" style="width:100%;">Upload</button>
```
Comment (cervenyprofiq) [inline fileUpload/index.php]: <button id="uploadBtn" type="submit" name="action" class="waves-effect waves-light btn" style="width:100%;" disabled>Upload</button>   // Add this for inicialize disabled button.

File: fileUpload/index.php (hunk 2 above)
Comment (petrvecera) [inline fileUpload/index.php]: This is mistake after refactor, that line should be in the .ready() function.

**Comment by petrvecera** [comment]
JS fix implemented by cervenyprofiq. RIP Git workflow

**Pull request closed without merging**